include/Song.h

```diff
@@ -101,14 +101,22 @@ class EXPORT Song : public TrackContainer
 	{
 		return m_nLoadingTrack;
 	}
+
 	inline int getMilliseconds() const
 	{
 		return m_elapsedMilliSeconds;
 	}
-	inline void setMilliSeconds( float ellapsedMilliSeconds )
```

Comment:
I prefer leaving `setMilliSeconds()` to removing, but it's not bad. We can add this again when needed.